### ISSUE TITLE
Select subtitle language

### ISSUE THREAD

**Issue by snagytx** [FaisalUmair/udemy-downloader-gui#90]
Hello, 

I see that subtitles were added [Fix search and add support for subtitles](https://github.com/FaisalUmair/udemy-downloader-gui/commit/cf9807e4f75d6de7f3232247ecaed0e22e5e1dd4) but there is no way to select which one to download. To me it seems that it download the first one available in the list.

It would be nice if it would select the one set in the tool? or maybe provide another option.

Thank you for the great work.

**Comment by FaisalUmair** [comment]
Yes you are right. It only downloads the first subtitle available. 

I had already planned for adding the option to chose subtitle. And I will surely add this option in next release.

**Comment by blackpanther989** [comment]
Another suggestion would be to allow the selection of no subtitles as well.

**Comment by bobbintb** [comment]
> Another suggestion would be to allow the selection of no subtitles as well.

I don't know it that was available at the time but this is an option now.

I would like to be able to select more than one subtitle, in addition to default. Many times I have to download twice to get all the English subtitles because some of the subtitles are in English and some are English(auto).
